lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -705,6 +705,169 @@ SPIRV::SPIRVInstruction *LLVMToSPIRV::transUnaryInst(UnaryInstruction *U,
                           BB);
 }
 
+/// Check if the module contains llvm.loop.* metadata
+static bool hasLoopMetadata(const Module *M) {
+  for (const Function &F : *M)
```

Comment:
I would love to see brackets in such cases, but it's up to you to apply this comment (as a comment from above about brackets).

Reply:
When starting out on my LLVM journey, I couldn't accept the braces' absence myself. But then came a review request to remove these... I'll gladly apply this suggestion 👍